XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1927,6 +1927,25 @@
               "Uri": "https://docs.microsoft.com/windows/uwp/design/style/spacing"
             }
           ]
+        },
+        {
+          "UniqueId": "CornerRadius",
+          "Title": "CornerRadius",
+          "Subtitle": "Round Corners",
+          "ImagePath": "ms-appx:///Assets/DefaultIcon.png",
+          "Description": "You may set CornerRadius property starting in WinUI 2.2. It sets round corners to your control",
+          "Content": "You may set CornerRadius property starting in Windows 10 1809. It sets round corners for your UI element.",
```

Comment:
I think those two sentences should not diverge. Something to point out regarding WinUI 2.2 is that that is the version that updated a lot of controls to use corner radius.